commit 1f9c29c3b399b19b75a12f7cb9086688f0131d1e (refs/changes/20/3820/1)
Author: Jason King <jason.king@joyent.com>
Date:   2018-04-14T12:31:45-05:00 (1 year, 6 months ago)
    
    OS-6891 usr/src/lib/fm is missing a dependency on libpcidb

diff --git a/usr/src/lib/Makefile b/usr/src/lib/Makefile
index 445238f973..7d337e7175 100644
--- a/usr/src/lib/Makefile
+++ b/usr/src/lib/Makefile
@@ -588,7 +588,8 @@ auditd_plugins: libbsm libsecdb libgss libmtmalloc
 brand:		libzonecfg libmapmalloc libipadm libcmdutils libproc librpcsvc
 cfgadm_plugins:	libdevice libdevinfo libhotplug librcm hbaapi libkstat libscf
 fm:		libexacct libipmi libzfs scsi libdevinfo libdevid libcfgadm \
-		libcontract libsysevent ../cmd/sgs/libelf libdladm libsff
+		libcontract libsysevent ../cmd/sgs/libelf libdladm libsff \
+		libpcidb
 $(SPARC_BLD)fm:	libpri
 gss_mechs/mech_dh: libgss
 gss_mechs/mech_dummy: libgss
